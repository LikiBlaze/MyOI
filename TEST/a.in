50 536870911
11009 5845 32610 14990 32703 3196 20486 3094 14344 30524 1588 29315 9504 7449 25201 13459 6619 20581 19797 14799 15282 19590 20799 28010 27158 20473 23623 18539 12293 6039 24180 18191 29658 7959 6192 19816 22889 19157 11512 16203 2635 24273 20056 20329 22647 26363 4887 18876 28434 29870 0 20143 23845 1417 21882 31999 10323 18652 10022 5700 3558 28477 27893 24390 5076 10713 2601 2511 21004 26870 17862 14689 13402 9790 15256 16424 5003 10586 24183 10286 27089 31427 28618 23758 9833 30933 4170 2155 25722 17190 19977 31330 2369 28693 21426 10556 3435 16550 7442 9513 0 0 30146 18061 21719 3754 16140 12424 16280 25997 16688 12530 22550 17438 19867 12950 194 23196 3298 20417 28287 16106 24489 16283 12456 25735 18115 11702 31317 20672 5787 12264 4314 24356 31186 20054 913 10809 1833 20946 4314 27757 28322 19559 23647 27983 482 4145 23197 20223 0 0 0 7130 2162 5536 20451 11174 10467 12045 21660 26293 26440 17254 20025 26155 29511 4746 20650 13187 8314 4475 28023 2169 14019 18788 9906 17959 7392 10203 3626 26478 4415 9315 25825 29335 25875 24373 20160 11834 28071 7488 28298 7519 8178 17774 32271 1764 2669 17193 0 0 0 0 13986 3103 8481 29214 7628 4803 4100 30528 2626 1544 1925 11024 29973 13062 14182 31004 27433 17506 27594 22726 13032 8493 143 17223 31287 13065 7901 19188 8361 22414 30975 14271 29171 236 30834 19712 25761 18897 4668 7286 12551 141 13695 2696 21625 28020 0 0 0 0 0 2126 26577 21695 22659 26303 17372 22467 4679 22594 23852 25485 1019 28465 21120 23153 2801 18088 31061 1927 9011 4758 32171 20316 9577 30228 12044 22759 7165 5110 7883 17087 29566 3488 29578 14475 2626 25628 5630 31929 25424 28521 6903 14963 124 24597 0 0 0 0 0 0 3738 13262 10196 32526 1265 8261 6203 8117 5031 20327 29012 30772 6412 25548 21154 21521 29791 14925 30189 21764 4941 20852 18663 13830 30901 17714 18959 17579 8366 13008 11478 1201 26059 6440 2304 12761 19358 2325 6478 5109 21114 14888 19802 22851 0 0 0 0 0 0 0 14461 22429 12994 27385 19406 6541 31112 28705 12836 32357 6073 29351 18824 14486 20557 23217 1627 9358 8527 13358 29338 23272 23870 29362 12897 13023 29618 10113 12718 18697 11586 24042 24424 24130 24230 4566 6560 8933 22297 29856 12054 16963 3585 0 0 0 0 0 0 0 0 29735 6655 16973 21458 14370 22533 2964 2608 2484 912 11636 10068 22849 4676 12939 2224 22143 23755 6512 22742 20176 21460 17826 3222 17871 1627 31935 15206 31784 23851 17399 22280 22702 12194 12735 1638 26535 5557 1994 10177 25706 6963 0 0 0 0 0 0 0 0 0 10549 15882 301 14414 16642 19856 24856 13143 11463 27612 30878 20425 32679 1753 18444 28297 12674 10041 9314 876 20073 12819 611 1018 14933 28113 30696 13170 23832 20041 26489 28686 19091 19498 2590 25991 15146 19354 19315 18652 26741 0 0 0 0 0 0 0 0 0 0 22045 11259 336 8760 11193 7606 25265 12182 28504 3830 23776 20609 29293 5998 17550 29557 25562 31628 6468 29542 26130 31241 27814 29175 20602 6078 20216 8684 8214 23993 25825 5602 23393 15760 2671 26429 28028 4085 10076 18787 0 0 0 0 0 0 0 0 0 0 0 15499 24971 6288 23848 32605 504 21222 22664 5707 2364 9011 22172 27490 18241 12165 25543 7620 20914 7592 6705 31819 9233 751 25206 4976 1540 304 11423 21099 11248 13585 13649 2972 17865 22914 11076 21546 28713 17547 0 0 0 0 0 0 0 0 0 0 0 0 18679 1770 15263 8520 13986 28290 15945 2866 18541 23246 25509 28319 27871 9602 28324 21133 24473 27153 25088 28571 29764 29902 17104 14424 3528 11601 26970 14016 5566 29 21544 25348 2089 2944 12638 22410 26464 5050 0 0 0 0 0 0 0 0 0 0 0 0 0 4682 1589 11343 609 32061 21222 1759 29955 20889 14147 691 7950 12844 21431 25621 749 27068 4537 20784 18036 32227 15186 7039 9854 25630 11225 15749 19924 3360 32258 24767 4945 14956 23319 32727 25412 21026 0 0 0 0 0 0 0 0 0 0 0 0 0 0 20356 31002 22550 9497 18585 9516 17965 23343 8076 17914 16143 31197 21949 25073 20427 14607 26174 24430 32405 6706 20627 29813 19376 30094 16566 16037 14737 29142 30815 5995 8257 6653 23937 30839 20483 1356 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 21016 1132 18231 17842 14626 2012 32638 4187 19691 1651 5663 21635 10894 10354 21417 13453 14009 7263 22234 5455 16304 16635 26304 14257 149 11125 12318 4214 27110 24029 29201 21081 21319 16859 24051 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 24156 31362 15265 11904 3677 29644 26910 14903 3562 28490 24949 1283 13654 30675 2221 5403 6924 3832 19370 3879 20260 19009 22620 23972 30004 21946 9782 26505 12393 32686 25314 6699 5590 12723 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 5939 19038 6411 31462 6235 12509 9962 3960 6494 1516 25270 24938 28870 59 14701 13972 26265 15118 16216 24556 7816 18331 3040 30213 29289 28083 1955 16086 20711 24485 24775 8381 29816 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 25952 6542 18116 1680 17111 25899 23074 789 23978 18133 29957 28690 26114 10009 12942 15791 1724 21364 29 25185 24779 7201 5072 1886 21975 1072 11334 22868 26154 14296 32169 20826 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 9677 15630 28651 2599 3310 4694 4687 30081 10117 12250 26668 1529 26680 7865 29422 8406 8827 6817 7517 27727 28667 29088 27682 19965 1341 5687 6022 11663 14722 6065 29310 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 20416 17903 29874 17125 23942 32746 31763 28424 27532 4807 12269 9319 5603 31908 24308 23482 1013 21137 26631 24115 26810 4085 23557 12291 21294 29997 29153 1055 25346 14709 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 249 7492 13713 5132 30115 16440 7959 24723 29705 6996 1053 25270 7480 18239 26424 27919 10867 17660 32499 28487 21197 7463 26634 12159 12023 1147 18393 13038 3926 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 10648 29459 6603 30808 4099 27831 29293 14601 7279 20800 18353 20449 13883 541 28316 4576 8763 9568 22337 18398 31419 19898 25829 13852 26817 24231 4450 16926 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 659 230 24521 10941 29561 15148 25163 1656 20676 793 22362 31755 16399 23147 8715 26947 21189 19570 8639 27664 15076 24516 11522 476 15616 20529 13235 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 12571 906 9465 19558 28963 14162 30525 12550 17470 331 31924 28351 14334 22926 10911 19738 16337 18338 1279 12394 7637 30715 28165 31592 19950 19136 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2506 13338 10005 16338 2624 28665 9971 25609 30569 19282 7086 9153 18374 28653 8195 9877 23827 28397 7573 6250 14641 29175 18820 944 32612 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1942 20290 8420 5566 3806 7586 16217 31451 1616 2610 11065 9167 16894 16075 13510 30301 19696 9574 15590 13162 31173 17969 27359 26032 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 26269 19427 28511 10423 30775 8780 30911 23553 4183 25392 15496 5765 875 1365 14903 28256 4461 475 6973 26822 26123 25548 21578 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 9790 25606 30196 27595 7951 16344 755 2482 1013 11673 23440 13429 5913 9763 5968 24409 4416 1909 17224 11760 26435 5205 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 29487 4320 959 5946 26807 8167 18701 11368 17693 21788 28533 30557 30975 3448 5022 18284 26223 29332 2377 28584 26949 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 12724 17983 22019 14777 9221 31112 32183 23857 19491 8926 4325 6487 19678 15970 11644 7535 15678 22669 31069 21992 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 32197 7784 16829 7728 29427 15872 10698 17613 18704 11028 11409 5546 9509 7186 30239 24238 26444 21314 22502 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 8851 25129 2112 23651 28150 11193 1455 15870 682 27466 20268 17714 19794 18635 25473 20973 22831 24902 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 28443 5178 13878 25771 703 14365 1382 16591 8824 2238 8024 32180 16596 20170 2328 12043 31311 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 28183 11059 7927 9488 1671 32529 5652 2259 7214 9861 25784 21287 2743 8611 4473 7129 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 18435 5842 20719 3504 14868 24866 10939 1882 9258 22751 28615 18599 28459 2662 26064 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 32757 20808 20279 19490 19436 6366 32076 7587 1387 7834 8361 13331 26049 8929 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 29493 12434 23841 6767 1736 19811 11600 11838 21893 31983 7329 29353 11370 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 31245 21795 6609 9253 11648 17433 9536 7209 3265 3498 23244 27650 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 22016 26842 190 16101 19813 30649 9524 19852 24475 28634 29892 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 27201 19855 9991 25698 4920 17781 22579 12932 32545 3341 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 13488 900 22526 8484 5539 7493 6194 28253 25012 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1561 15835 1841 31498 2786 18530 31541 18806 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 28792 13393 13211 13550 21579 26980 30972 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 9278 74 20194 1621 21498 13827 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 31277 19791 6583 13579 11160 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 20419 26490 160 3450 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 12925 9073 10381 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 27009 27968 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 10209 