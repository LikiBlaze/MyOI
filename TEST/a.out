648082380258 479665660